app/assets/stylesheets/static.scss

```diff
@@ -15,3 +19,20 @@
     font-size: 1.5em;
   }
 }
+
+.button, button, input[type='button'], input[type='reset'], input[type='submit'] {
+  background-color: #C09AC1;
+  border-color:  #C09AC1;
+}
+
+a {
```

Comment:
Usually you want to add classes to the these objects so we aren't changing things so broadly, in this case I can totally understand why you wanted it this way.

Reply:
The app is prettttty small, so, for the sake of fluidity, it seemed okay this time around.